packages/core/src/components/navigation/NavbarSlider/NavbarSlider.tsx

```diff
@@ -37,15 +51,15 @@ function NavbarSlider() {
           title="Go to FastStore home"
           data-fs-navbar-slider-logo
         >
-          <Logo />
+          <Logo loading="eager" {...logo} />
```

Comment:
we don't need eager loading here since this image is under de `NavbarSlider` (`SlideOver` component) and it isn't in the viewport in the first render.

Reply:
Nice, I forgot this one it's just for the mobile version 👍🏼